### PR TITLE
Fixed InfoPanel layout in Trash

### PR DESCRIPTION
**Pull request by kazup01**
# White
screen shot 0029-08-06 at 12 58 22 pm

# Dark
screen shot 0029-08-06 at 12 58 46 pm

refs: #765

### CODE REVIEW

**Comment by asmsuechan** [comment]
`InfoPanelTrashed` Umm :thinking:

**Comment by asmsuechan** [comment]
I'll refactor this later because this PR contains important change regarding https://github.com/BoostIO/Boostnote/pull/765.

File: browser/main/Detail/InfoPanelTrashed.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      <div styleName='group-section-control'>
+        {storageName}
+      </div>
+    </div>
+    <div styleName='group-section'>
+      <div styleName='group-section-label'>
+        Folder
+      </div>
+      <div styleName='group-section-control'>
+        <text>in Trash</text>{folderName}
```
Comment (asmsuechan) [inline browser/main/Detail/InfoPanelTrashed.js]: I prefer `Trash` than `in Trash` in this case.

Comment (kazup01) [inline browser/main/Detail/InfoPanelTrashed.js]: Fixed!

**Comment by kazup01** [comment]
Thanks!

**Pull request merged**